assets/js/directives/contextual-message.directive.js

```diff
@@ -37,10 +37,13 @@ function contextualMessage($cookies, $window, Wallet, SecurityCenter, filterFilt
     };
 
     scope.shouldShow = () => {
-      let balance   = Wallet.total('');
-      let security  = SecurityCenter.security.score;
-      let isTime    = scope.msgCookie ? Date.now() > scope.msgCookie.when : true;
-      return balance > 20000000 && security < 0.5 && isTime;
+      let isTime = scope.msgCookie ? Date.now() > scope.msgCookie.when : true;
+      let hasBalance    = Wallet.total('') > 0;
+      let didBackup     = Wallet.status.didConfirmRecoveryPhrase;
+      let has2FA        = Wallet.settings.needs2FA;
+      let verifiedEmail = Wallet.user.isEmailVerified;
+
+      return isTime && hasBalance && !didBackup && !(has2FA || verifiedEmail);
```

Comment:
has2FA is can be null / undefined while loading this data. Balance is loaded separately. We should be careful not to assume a load order. Currently && means this won't be a problem, but a few test cases can make sure we don't break it later.